lib/morph-cli.rb

```diff
@@ -97,17 +97,15 @@ def self.in_directory(directory)
   end
 
   def self.create_tar(directory, paths)
-    tempfile = File.new('/tmp/out', 'wb')
+    File.new('/tmp/out', 'wb')
```

Comment:
this should probably use a random-ised file name, at least while writing the output, and then mv to it's final location after